### PR TITLE
[FIX] synchronize.py: Delete temporal folder

### PR DESCRIPTION
**Pull request by JesusZapata**

### CODE REVIEW

File: synchronize.py [hunk 1]
```diff
@@ -127,7 +127,18 @@ def create_component(self, project, branch):
         try:
             print subprocess.check_output(cmd)
         except subprocess.CalledProcessError:
-            logger.error('WeblateAPI.create_component > Error processing the project %s on branch %s' % (project['slug'], branch['branch_name']))
+          logger.error('WeblateAPI.create_component > Error processing the project %s on branch %s' % (project['slug'], branch['branch_name']))
```
Comment (moylop260) [inline synchronize.py:130]: use 4 spaces

Comment (moylop260) [inline synchronize.py:130]: `logger` don't need `%`
Use `,` instead.

Comment (JesusZapata) [reply synchronize.py:130]: @moylop260 
Thanks for you comment
I register this PR https://github.com/Vauxoo/SynRunbotWeblate/pull/5

Comment (JesusZapata) [reply synchronize.py:130]: @moylop260 
I did not know it could be done, thanks
On this PR is fixed https://github.com/Vauxoo/SynRunbotWeblate/pull/5

**Pull request merged**